src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
@kpayson64: I don't have any particular thoughts here and I'm fairly certain that the two of you are both more familiar with the details than I am.